src/test/groovy/org/prebid/server/functional/testcontainers/PbsServiceFactory.groovy

```diff
@@ -51,4 +53,8 @@ class PbsServiceFactory {
             containers.remove(key)
         }
     }
+
+    private static int containersCount() {
```

Comment:
name it like it was named: getContainersCount(). Then you can call that method just like 'containersCount'.